Add a note regarding new security restriction (#36693)
Azure/azure-content: articles/active-directory-b2c/active-directory-b2c-ui-customization-custom.md

2. Paste the copied snippet in a text editor, and then save the file as *customize-ui.html*.

> > [!NOTE] HTML form elements will be removed due tosecurity restrictions If you use login.microsoftonline.com. Please use b2clogin.com if you want to use HTML form elements in your custom HTML content. Please see [Use b2clogin.com](b2clogin.md) for other benets.

Reviewer: ```suggestion
> [!NOTE] HTML form elements will be removed due to security restrictions if you use login.microsoftonline.com. Please use b2clogin.com if you want to use HTML form elements in your custom HTML content. See [Use b2clogin.com](b2clogin.md) for other benefits.
```

2. Paste the copied snippet in a text editor, and then save the file as *customize-ui.html*.

> [!NOTE]

> > HTML form elements will be removed due to security restrictions. If you use login.microsoftonline.com, use b2clogin.com if you want to use HTML form elements in your custom HTML content. See [Use b2clogin.com](b2clogin.md) for other benefits.

Reviewer: ```suggestion
> HTML form elements will be removed due to security restrictions if you use login.microsoftonline.com. Please use b2clogin.com if you want to use HTML form elements in your custom HTML content. See [Use b2clogin.com](b2clogin.md) for other benefits.
```

Reviewer 2: Thanks for catching that. I made another revision.

> > HTML form elements will be removed due to security restrictions, if you use login.microsoftonline.com. Please use b2clogin.com if you want to use HTML form elements in your custom HTML content. See [Use b2clogin.com](b2clogin.md) for other benefits.

Reviewer: ```suggestion
> HTML form elements will be removed due to security restrictions if you use login.microsoftonline.com. Please use b2clogin.com if you want to use HTML form elements in your custom HTML content. See [Use b2clogin.com](b2clogin.md) for other benefits.
```
